pages.fr/common/mongo.md

```diff
@@ -0,0 +1,20 @@
+# mongo
+
+> Client shell pour MongoDB.
+> Plus d'informations: <https://docs.mongodb.com/manual/reference/program/mongo>.
+
+- Connecte à une base de données (database) :
+
+`mongo {{nom_de_la_base_de_données}}`
```

Comment:
Is `base de données` a good translation?

Reply:
For the word `database`, `base de données` is the correct translation :D 
[you can check here](https://fr.techdico.com/traduction/anglais-francais/database.html)

Reply:
Then why `{{nom_de_la_base_de_données}}`? I assume this means `{{database_name}}`, but the English page just uses `{{database}}`.

Reply:
Well, technically you're right and most of the french devs will understand.
But if I only set `base de données` it does not sound right.
Because "database" for us refers to the object itself, not his name.

Like in english, if you need to set a username, you won't ask to someone "Please insert your user"

And that kind the same thing in french for `base de données` ^^

PS: Yes French sucks